<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>index</class>
 <widget class="QWidget" name="index">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f1f2f6;
</string>
  </property>
  <widget class="QPushButton" name="radio_btn_right">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>150</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2f3542;
border: 3px solid #2f3542;
border-radius: 10px;
font-size: 150px;
font: &quot;Comic Sans MS&quot;;
color: #f1f2f6;
text-align: center;</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="radio_btn_left">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>150</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2f3542;
border: 3px solid #2f3542;
border-radius: 10px;
font-size: 150px;
font: &quot;Comic Sans MS&quot;;
color: #f1f2f6;
text-align: center;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="volume_slider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>724</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
border: 2px solid #2f3542;
background: #f1f2f6;
height: 25px;
border-radius: 10px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1, stop: 0 #ffffff, stop: 1 #70a1ff);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, stop: 0 #70a1ff, stop: 1 #3742fa);
border: 2px solid #2f3542;
border-radius: 10px;
}

QSlider::handle:horizontal {
background: #2f3542;
border: 3px solid #2f3542;
width: 25px;
margin-top: -4px;
margin-bottom: -4px;
border-radius: 5px;
}
</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_btn_minus">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>540</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #70a1ff;
border: 3px solid #1e90ff;
border-radius: 10px;
font-size: 100px;
font: &quot;Comic Sans MS&quot;;
color: whitesmoke;
padding-bottom: 15px;
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_btn_plus">
   <property name="geometry">
    <rect>
     <x>879</x>
     <y>540</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #70a1ff;
border: 3px solid #1e90ff;
border-radius: 10px;
font-size: 50px;
font: &quot;Comic Sans MS&quot;;
color: whitesmoke;
padding-bottom: 8px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mute_btn">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>460</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #dfe4ea;
border: 3px  solid #a4b0be;
border-radius: 15px;
font-size: 20px;
color: #2f3542;
font: &quot;MV Boli&quot;;</string>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>40</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #70a1ff;
border: 3px solid #1e90ff;
border-radius: 10px;
font: &quot;Comic Sans MS&quot;;
color: whitesmoke;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="radio_icon_btn">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>150</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #70a1ff;
border: 3px solid #1e90ff;
border-radius: 10px;
font: &quot;Comic Sans MS&quot;;
color: whitesmoke;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
